lib/Github/HttpClient/Plugin/Authentication.php

```diff
@@ -46,62 +46,26 @@ public function __construct($tokenOrLogin, $password, $method)
      * {@inheritdoc}
      */
     public function handleRequest(RequestInterface $request, callable $next, callable $first): Promise
+    {
+        $request = $request->withHeader(
+            'Authorization',
+            $this->getAuthorizationHeader()
+        );
+
+        return $next($request);
+    }
+
+    private function getAuthorizationHeader()
```

Comment:
Add a `string` return type for this method

Reply:
Do you want a genuine return type, or just phpdoc. Let me know, and I will add the one you want. :)

Reply:
I would add a "real" return type here